src/components/SingleArticle.jsx

```diff
@@ -36,6 +39,35 @@ const SingleArticle = () => {
         setSingleArticle(singleArticleCopy)
         decreaseVotes(article_id)
     }
+
+    const handleChange = (event) => {
+        setAddComment(event.target.value);
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+
+        const newComment = {
+            comment_id: Date.now(),
+            author: "cooljmessy",
+            body: addComment,
+            created_at: new Date().toISOString(),
+            votes: 0
+        }
+        setComments((currComments) => {
+            return [newComment, ...currComments]
+            })
+
+        postNewComment(article_id, {username: "cooljmessy", body: addComment})
+        .then((response)=> {
+            setAddComment("")
+        })
+        .catch((err) => {
+            <p>Sorry, there was an error, please try again.</p>
```

Comment:
We don't seem to be getting this error message when we make an online post. I'm thinking you may need to possibly have an error state, then when it's true the p tag shows. Have a play around with it though and see how you get on!